docker/release/centos7/Dockerfile

```diff
@@ -1,36 +1,24 @@
-FROM centos:7
-MAINTAINER Andrew Schwartzmeyer <andschwa@microsoft.com>
+# Docker image file that describes an Centos7 image with PowerShell installed from Microsoft YUM Repo
 
-ARG POWERSHELL_RELEASE=v6.0.0-alpha.16
-ARG POWERSHELL_PACKAGE=powershell-6.0.0_alpha.16-1.el7.centos.x86_64.rpm
+FROM centos:7
+MAINTAINER Raghu Shantha <raghus@microsoft.com>
```

Comment:
@joeyaiello could you please comment on this question? This PR looks good except for this question.